bin/common/install_tools.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset
+
+# Get version information and set destination dirs
+. "$(dirname "$0")/versions.sh"
+
+# Installs tools needed to build and run HCF
+SCF_BIN_DIR="${SCF_BIN_DIR:-output/bin}"
+cf_url="${cf_url:-https://cli.run.pivotal.io/stable?release=linux64-binary&version=${CFCLI_VERSION}&source=github-rel}"
+kubectl_url="${kubectl_url:-https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl}"
+k_url="${k_url:-https://github.com/aarondl/kctl/releases/download/v${K_VERSION}/kctl-linux-amd64}"
+kk_url="${kk_url:-https://gist.githubusercontent.com/jandubois/40a5b3756cf4bcbed940e6156272c0af/raw/${KK_VERSION}/kk}"
+helm_url="${helm_url:-https://kubernetes-helm.storage.googleapis.com/helm-v${HELM_VERSION}-linux-amd64.tar.gz}"
+
+mkdir -p "${SCF_BIN_DIR}"
+
+SCF_BIN_DIR="$(cd "${SCF_BIN_DIR}" && pwd)"
+
+echo "Fetching cf CLI $cf_url ..."
+wget -q "$cf_url" -O "/tmp/cf.tgz"
+
+echo "Unpacking cf CLI ..."
+tar -xzf "/tmp/cf.tgz" -C "${SCF_BIN_DIR}"
+
+wget -q "${k_url}" -O "${SCF_BIN_DIR}/k"
+wget -q "${kk_url}" -O "${SCF_BIN_DIR}/kk"
+
+echo "Fetching helm from ${helm_url} ..."
+wget -q "${helm_url}" -O - | tar xz -C "${SCF_BIN_DIR}" --no-same-owner --strip-components=1 linux-amd64/helm
+
+echo "Making binaries executable ..."
+chmod a+x "${SCF_BIN_DIR}/cf"
+chmod a+x "${SCF_BIN_DIR}/k"
+chmod a+x "${SCF_BIN_DIR}/kk"
+chmod a+x "${SCF_BIN_DIR}/helm"
```

Comment:
Hmm.. looks like we're "missing"(?) the wget/curl for kubectl also. I think this was generally because kubectl comes in through the suse packages.. since this script isn't generally installing kubernetes, I'm not convinced it should be installing kubectl (but we should probably remove `kubectl_url` if not).

If we do want it to install kubectl, it can do so optionally only if `kubectl` isn't already in PATH somewhere (which will prevent it from installing kubectl over the package-installed kubectl for deployments from our packer-built image)